src/main/java/net/sknv/qooq/Main.java

```diff
@@ -1,25 +1,50 @@
 package net.sknv.qooq;
 
 import net.dv8tion.jda.api.JDABuilder;
+import net.dv8tion.jda.api.entities.Message;
 import net.dv8tion.jda.api.entities.MessageChannel;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.sknv.qooq.cards.Blackjack;
 
 import javax.annotation.Nonnull;
 import javax.security.auth.login.LoginException;
 
 public class Main extends ListenerAdapter {
 
+    Blackjack bj;
+
     public static void main(String[] args) throws LoginException {
-        //https://discord.com/api/oauth2/authorize?client_id=714593234285756517&scope=bot&permissions=8
-        new JDABuilder(Utils.getToken()).addEventListeners(new Main()).build();
+        // Invite bot to server w/ admin permissions
+        // https://discord.com/api/oauth2/authorize?client_id=714593234285756517&scope=bot&permissions=8
+        JDABuilder.createDefault(Utils.getToken()).addEventListeners(new Main()).build();
+    }
+
+    public Main() {
+        bj = new Blackjack();
     }
 
     @Override
     public void onMessageReceived(@Nonnull MessageReceivedEvent event) {
         MessageChannel channel = event.getChannel();
-        if (!event.getAuthor().isBot()) {
-            channel.sendMessage("Sogres is a cook").queue();
+        Message msg = event.getMessage();
+        if (msg.getContentRaw().startsWith("kek ")) {
+            parse(msg);
+        }
+    }
+
+    private void parse(Message msg) {
+        String text = msg.getContentRaw();
+        String[] params = text.split(" ");
+        switch (params[1]) {
+            case "asd":
```

Comment:
maybe remove the debug command?

Reply:
left as example/placeholder since none of the commands actually do anything rn because the bj game is not developed yet